app/models/product.rb

```diff
@@ -1,3 +1,3 @@
 class Product < ApplicationRecord
-  scope :recent, -> {order(created_at: :desc)}
+  scope :recent, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, -> {order(created_at: :desc)}
```
Thêm 1 space trước block {}